SwiftCompilerSources/Sources/Optimizer/FunctionPasses/CopyToBorrowOptimization.swift

```diff
@@ -323,7 +326,12 @@ private func createEndBorrows(for beginBorrow: Value, atEndOf liverange: Instruc
   //   destroy_value %2
   //   destroy_value %3  // The final destroy. Here we need to create the `end_borrow`(s)
   //
-  for endInst in collectedUses.allLifetimeEndingInstructions {
+  var worklist = collectedUses.allLifetimeEndingInstructions
```

Comment:
Given that the return value of `allLifetimeEndingInstructions` is not a simple Array anymore, but a worklist which has to be `deinitialize`d, it's probably simpler to remove the `var allLifetimeEndingInstructions` and inline the worklist construction directly here.

Reply:
Done. Thanks